config/settings/base.py

```diff
@@ -212,8 +212,23 @@
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default="django://")
 # https://docs.celeryproject.org/en/stable/userguide/optimizing.html#broker-connection-pools
 # https://docs.celeryq.dev/en/latest/userguide/optimizing.html#broker-connection-pools
+# Configured to 0 due to connection issues https://github.com/celery/celery/issues/4355
 CELERY_BROKER_POOL_LIMIT = env(
-    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=1000)
+    "CELERY_BROKER_POOL_LIMIT", default=env("CELERY_BROKER_POOL_LIMIT", default=0)
+)
+# https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-heartbeat
+CELERY_BROKER_HEARTBEAT = env.int(
+    "CELERY_BROKER_HEARTBEAT", default=env("CELERY_BROKER_HEARTBEAT", default=0)
+)
+# https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-broker_connection_max_retries
+CELERY_BROKER_CONNECTION_MAX_RETRIES = env.int(
+    "CELERY_BROKER_CONNECTION_MAX_RETRIES",
+    default=env("CELERY_BROKER_CONNECTION_MAX_RETRIES", default=0),
+)
+# https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-channel-error-retry
+CELERY_BROKER_CHANNEL_ERROR_RETRY = env.bool(
+    "CELERY_BROKER_CHANNEL_ERROR_RETRY",
+    default=env("CELERY_BROKER_CHANNEL_ERROR_RETRY", default=True),
```

Comment:
Every environment variable is duplicated

Reply:
fixed